insights-client.spec

```diff
@@ -133,11 +139,15 @@ sed -i '/### Begin insights-client-ros ###/,/### End insights-client-ros ###/d;/
 %{_sysconfdir}/insights-client/rpm.egg*
 %{_bindir}/*
 %{_unitdir}/*
-%{_presetdir}/*
 %attr(444,root,root) %{_sysconfdir}/insights-client/*.pem
 %attr(444,root,root) %{_sysconfdir}/insights-client/redhattools.pub.gpg
-%{_defaultdocdir}/%{name}
 %{python3_sitelib}/insights_client/
+%{_defaultdocdir}/%{name}
```

Comment:
Shouldn't we also try to create this directory: `%{_defaultdocdir}/%{name}` in spec file?

Reply:
AFAIK, rpm creates it automatically